tests/Doctrine/Tests/Common/Persistence/Mapping/SymfonyFileLocatorTest.php

```diff
@@ -62,6 +62,90 @@ public function testGetAllClassNames()
         $this->assertEquals(array("Foo\\stdClass"), $locator->getAllClassNames("global"));
     }
 
+
+    public function customNamespaceSeparatorProvider()
+    {
+        return array(
+            'directory separator' => array(DIRECTORY_SEPARATOR, "/_custom_ns/dir"),
+            'underscore separator' => array('_', "/_custom_ns/underscore"),
```

Comment:
I've re-read PSR-0 again and realised that underscore notation should affect only `Class_Name` notation and should not affect the `name_space` part, so it's indeed a bad setup example, but not sure if the should forcibly prohibit underscores for use